doc/menu_lua_api.txt

```diff
@@ -79,6 +79,8 @@ core.explode_table_event(string) -> table
 core.explode_textlist_event(string) -> table
 ^ returns e.g. {type="CHG", index=1}
 ^ type: "INV" (no row selected), "CHG" (selected) or "DCL" (double-click)
+core.set_formspec_prepend(formspec)
+^ string to be added to every formspec, to be used for theming.
```

Comment:
This prepend is not added to ingame fromspecs, is it?
```suggestion
^ string to be added to every mainmenu formspec, to be used for theming.
```

Reply:
lol, I'll clarify that